uitests/plugins/org.jboss.tools.fuse.reddeer/src/org/jboss/tools/fuse/reddeer/wizard/NewFuseIntegrationProjectWizard.java

```diff
@@ -184,12 +184,19 @@ public void selectTemplate(String... path) {
 	public List<String> getAllAvailableTemplates() {
 		log.debug("Retrieving available templates");
 		List<String> templates = new ArrayList<String>();
-		new RadioButton("Use a predefined template").toggle(true);
+		new RadioButton(this, "Use a predefined template").toggle(true);
 		for (TreeItem item : new DefaultTree().getAllItems()) {
 			if (item.getItems().isEmpty()) {
 				templates.add(item.getText());
 			}
 		}
 		return templates;
 	}
+	
+	/**
+	 * Click on BTN 'Verify' project Camel version
+	 */
+	public void verifyBTN() {
```

Comment:
I would prefer to use a different name according the functionality --> clickVerifyButton suits me better :-).